packages/backend/src/telemetry-notification.ts

```diff
@@ -0,0 +1,80 @@
+import fs from 'node:fs/promises';
+import os from 'node:os';
+import path from 'node:path';
```

Comment:
This is compatible with the node runtime only - it would break any apps not running on node. We need to make this logic isomorphic or think of a different way to show the telemetry notice.

For now, one other suggestion would be to simply show a shorter log (without the border) every time.